android/build.gradle

```diff
@@ -144,6 +146,7 @@ android {
         targetSdkVersion 30
         versionCode  System.getenv("VERSIONCODE").toInteger()
         versionName  System.getenv("SHORTVERSION")
+        buildConfigField "String", "ADJUST_SDK_TOKEN", '"' + System.getenv("ADJUST_SDK_TOKEN") + '"'
```

Comment:
I mean if anybody does not have the ADJUST_SDK_TOKEN and does a release build this feature will just silently broken. Can we maybe get an assert in the gradle-release task or in the android_package.sh that a release build requires this needs to be set? :)

Reply:
My idea was to make it possible to build for release without having adjust active for testing purposes. But maybe we do not need that.

Reply:
we only use the release-job builds for the Playstore so they should be including everything